cmd/clusterawsadm/cmd/ami/ami.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ami
+
+import (
+	"github.com/spf13/cobra"
+	cp "sigs.k8s.io/cluster-api-provider-aws/cmd/clusterawsadm/cmd/ami/copy"
+)
+
+// RootCmd is the root of the `ami command`
+func RootCmd() *cobra.Command {
+	newCmd := &cobra.Command{
+		Use:   "ami [command]",
+		Short: "AMI commands",
+		Args:  cobra.NoArgs,
+		Long:  "All AMI related actions",
```

Comment:
Might be nice to have a more desciptive description even if its acting as a container for subcommands.

Reply:
Yeah, I agree here. It's ok to reuse text constants/vars and repeat documentation.
After the clusterawsadm v2 rewrite, I've tended towards long explanations of the commands.